StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -95,7 +95,8 @@ final class PaymentSheetLoader {
                 analyticsClient.logPaymentSheetLoadSucceeded(
                     loadingStartDate: loadingStartDate,
                     linkEnabled: isLinkEnabled,
-                    defaultPaymentMethod: paymentOptionsViewModels.stp_boundSafeObject(at: defaultSelectedIndex)
+                    defaultPaymentMethod: paymentOptionsViewModels.stp_boundSafeObject(at: defaultSelectedIndex),
+                    intent: intent
```

Comment:
nit: Feels a little awkward to send the entire intent, cause I'm worried we may end up logging more than we should-- maybe we change the signature of `logPaymentSheetLoadSucceeded` to accept:
`intentAnalyticsValue: String` instead?